Makefile

```diff
@@ -161,12 +168,12 @@ deploy-role: deploy-namespace
 deploy-postgres: deploy-namespace
 	python3 ./tools/deploy_postgres.py --namespace "$(NAMESPACE)" --profile "$(PROFILE)" --target "$(TARGET)"
 
-jenkins-deploy-for-subsystem:
-	export TEST_FLAGS=--subsystem-test && export ENABLE_AUTH="True" && $(MAKE) deploy-all
+jenkins-deploy-for-subsystem: build-dummy-ignition-image
+	export TEST_FLAGS=--subsystem-test && export ENABLE_AUTH="True" && export DUMMY_IGNITION=${DUMMY_IGNITION} && $(MAKE) deploy-all
```

Comment:
export DUMMY_IGNITION=minikube-local-registry/ignition-dummy-generator:minikube-test

Reply:
@ronniel1 kubernetes will still treat it as external and will try to connect